src/Trading/hooks/form.ts

```diff
@@ -25,6 +25,12 @@ function getSpendableBalanceWithoutBaseReserve(accountMinimumBalance: BigNumber,
   return spendableBalance.cmp(BigNumber(0)) < 0 ? BigNumber(0) : spendableBalance
 }
 
+// replaces ',' with '.' in a string
+// this can be used with strings that represent a number before passing them to Big()
+export function replaceCommaWithDot(input: string) {
+  return input.replace(/,/g, ".")
```

Comment:
Is there anything we can do to prevent accidents in the case that someone tries to enter 3000 as `3,000`?

Bonus question: Does the form do its job if I enter 0.25 as `.25`?

Reply:
One more thing:
Now is logic is custom for the trading view. Actually, it should be consistent across all our forms… 🙈

Reply:
Regarding the bonus question: yes it would work if we change the regex pattern. Right now it will expect something like `x.xx` but seems like Big.js can also use strings without leading 0.
And I don't think we can do much about the `3,000` meaning 3000 or at least not without much effort. Seems like this locale thing is quite complex and to be honest I still don't fully understand why iOS is causing this issue anyways.

Reply:
I added the additional handling to the payment form as well, and I think the trading and payment form would be the only places where we actually expect inputs with fractions? So it should be fine, but I see your point that we could spend more efforts to fully prevent this issue in the future.